CONTRIBUTING.md

```diff
@@ -59,6 +59,18 @@ This catalog is documented on the [kpt website]. Follow the
 Changes to other documentation such as examples and README files can follow the
 same pull request format as code changes.
 
+## How to Release
+
+We have spearate a release process for each language, golang and typescript.
+All functions written in the language get released together. Maintainers can
+create releases through the Github UI or through the git CLI. Follow the
+commands below to release using the git CLI.
+
+```sh
+git tag release-ts-functions-v0.0.5
```

Comment:
This is an anti-pattern.

Releases should be done through the UI to have a GitHub release associated with it.

Reply:
Removed. Recommend only releasing through UI now.